deployments/deployments.go

```diff
@@ -0,0 +1,77 @@
+// Package deployments provides functions for decoding V2 paasta deployments
+// of the form:
+//   "v2": {
+//    "deployments": {
+//		"everything": {
+//		  "docker_image": "services-fluffy:paasta-abcdfff",
+//		  "git_sha": "abcdfff"
+//		},
+//		"stagef": {
+//		  "docker_image": "services-fluffy:paasta-abcdfff",
+//		  "git_sha": "abcdfff"
+//		}
+//	  },
+//  }
+package deployments
+
+import (
+	"fmt"
+
+	paastaconfig "github.com/Yelp/paasta-tools-go/config"
+)
+
+const (
+	dockerRepo = "docker-paasta.yelpcorp.com:443"
```

Comment:
don't hardcode this. there is another system-paasta-config you can get it from. I'd parameterize `getImageURL` with a docker registry address

Reply:
Using the docker registry file now.